Gopkg.toml

```diff
@@ -19,15 +19,15 @@ ignored = ["github.com/karalabe/hid", "github.com/satori/go.uuid", "github.com/i
 
 [[override]]
   name = "github.com/RTradeLtd/database"
-  version = "v1.1.1"
```

Comment:
going forward, i wonder if we should pin this down to `v1.1.*` (if `dep` even allows version globs like that), since based on our versioning patch increments should maintain full compatibility

Reply:
I believe with dep we can use constraints like `version = ^1.1.1` or something like that

Reply:
this might be a good idea, to avoid having to manually change it every time